sonar-ws/src/main/java/org/sonarqube/ws/client/project/ProjectsService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonarqube.ws.client.project;
+
+import org.sonarqube.ws.client.BaseService;
+import org.sonarqube.ws.client.PostRequest;
+import org.sonarqube.ws.client.WsConnector;
+
+/**
+ * Maps web service {@code api/projects}.
+ * @since 5.5
+ */
+public class ProjectsService extends BaseService {
+
+  public ProjectsService(WsConnector wsConnector) {
+    super(wsConnector, "api/projects");
+  }
+
+  /**
+   * Provisions a new project. This method does not allow to specify the parameter {@code branch}.
+   *
+   * @param projectKey the non-null key of project
+   * @param projectName the non-null name of project
+   * @throws org.sonarqube.ws.client.HttpException if HTTP status code is not 2xx.
+   */
+  public void create(String projectKey, String projectName) {
+    PostRequest request = new PostRequest(path("create")).setParam("key", projectKey).setParam("name", projectName);
+    call(request);
+  }
+
+  /**
+   * Deletes a project with the specified id.
+   *
+   * @throws org.sonarqube.ws.client.HttpException if HTTP status code is not 2xx.
+   */
+  public void deleteById(String projectId) {
```

Comment:
we said we didn't want to have a documentation on the ws client side. We don't want business logic, and it's a way to introduce business logic. The documentation is the Web Api.
If we had to introduce business logic, I would much prefer a Builder pattern.
I'm ok to have the String projectId instead of the DeleteWsRequest for the time being. I'd like a few braking changes in forward versions as possible. In this case, it will be if we add in a later version a DeleteWsRequest